identity-core/Cargo.toml

```diff
@@ -11,6 +11,7 @@ repository = "https://github.com/iotaledger/identity.rs"
 description = "The core traits and types for the identity-rs library."
 
 [dependencies]
+base64 = { version = "0.13", default-features = false, features = ["std"] }
```

Comment:
Depends on whether or not we want parity with `RevocationList2020` in the future, since `multibase` uses a sentinel character prefix to indicate the encoding.

https://github.com/multiformats/multibase#multibase-table

In this case `base64url` with no padding would be prefixed with `u`.

Reply:
> Depends on whether or not we want parity with RevocationList2020

I think since we use roaring bitmap, our revocation method will not be compatible with `RevocationList2020` anyway, if that's what you mean. But maybe I misunderstand.

> In this case base64url with no padding would be prefixed with u.

In that case, I don't think we want to change our spec to multibase since it doesn't seem to have a particular advantage for our case (?), so I guess we can leave this as-is.

Reply:
That's news to me then, I was still under the impression we were going with a compressed bitstring serialization with a roaring bitmap internal representation.

> In that case, I don't think we want to change our spec to multibase since it doesn't seem to have a particular advantage for our case (?), so I guess we can leave this as-is.

The benefit of `base64` over `multibase` is better interoperability since Base64 is more common, but implementers will need `multibase` regardless since it's used in `VerificationMethod`.

The other argument is fitting the [Data URI scheme](https://en.wikipedia.org/wiki/Data_URI_scheme). I originally used `data:,` since we don't need to decode the contents and so the media type defaults to `text/plain;charset=US-ASCII`. We could instead use something like `data:application/octet-stream;base64,` and then use proper `base64` encoding for the bitmap.

Reply:
https://github.com/iotaledger/identity.rs/pull/861#issuecomment-1149822713